src/main/resources/swagger/api-docs.yaml

```diff
@@ -3240,7 +3240,6 @@ definitions:
 
   NIHStatus:
     required:
-      - loginRequired
```

Comment:
so this was actually never used. maybe it was used at one point? the new logic shouldn't need it.

when you link, the only fields we set in thurloe are `linkedNihUsername` and `linkExpireTime`. `linkExpireTime` will be now + 30 days. 

when the whitelist syncs every 6 hours, it will filter out and remove all linked accounts that were linked > 30 days ago.

the UI will display the "you must re-link within 24 hours" banner once you reach the 29th day.

if there is no linkExpireTime set, you will be removed upon the next whitelist sync. this should never happen but there are safegaurds in place to make it work